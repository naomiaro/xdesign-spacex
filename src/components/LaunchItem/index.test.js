import React from 'react';
import ReactDOM from 'react-dom';
import { LaunchItem } from './index';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <LaunchItem
      flightNumber={1}
      missionName={'FalconSat'}
      rocketName={'Falcon 1'}
      launchDateUTC={'2006-03-24T22:30:00.000Z'}
    />,
    div
  );
});

it('Renders a LaunchItem', () => {
  const wrapper = shallow(
    <LaunchItem
      flightNumber={1}
      missionName={'FalconSat'}
      rocketName={'Falcon 1'}
      launchDateUTC={'2006-03-24T22:30:00.000Z'}
    >
      Refresh
    </LaunchItem>
  );
  expect(wrapper).toMatchSnapshot();
});
