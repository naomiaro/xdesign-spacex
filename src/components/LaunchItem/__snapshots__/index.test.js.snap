// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a LaunchItem 1`] = `
<li
  className="LaunchItem"
>
  <div
    className="flightNumber"
  >
    #
    1
  </div>
  <div
    className="missionName"
  >
    FalconSat
  </div>
  <div
    className="info"
  >
    <div
      className="date"
    >
      25th Mar 2006
    </div>
    <div
      className="rocketName"
    >
      Falcon 1
    </div>
  </div>
</li>
`;
